blog/2024-08-29-updated-roadmap.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Updated Roadmap
```

Comment:
Why not something like "The Road to 1.0"?

Without the 1.0 context this post feels a bit lacking.

Reply:
Yeah agree, updated the title and added an image for the blog.